api/controllers/HomeController.js

```diff
@@ -8,7 +8,8 @@
 module.exports = _.mapValues({
   async index (req, res) {
     const boxes = await Box.find({owner: req.user.name});
-    res.view('home/view', {boxes});
+    const prefs = await UserPreferences.find({user: req.user.name});
```

Comment:
Ok, we can keep this one as it is, but let's try to use the existing API as much as possible for future stuff (e.g. viewing other users' boxes, since there are actually security concerns for those).